src/main/java/com/greenfox/lvlup/model/dto/library/BadgeDTO.java

```diff
@@ -7,16 +7,16 @@
 import java.util.ArrayList;
 import java.util.List;
 
-@JsonSerialize
+//@JsonSerialize
```

Comment:
If it's in a comment, you probably don't need it! :)

Reply:
Deleted commented lines from BadgeDTO class